platform/android/java/lib/src/org/godotengine/godot/vulkan/VkRenderer.kt

```diff
@@ -73,11 +71,7 @@ internal class VkRenderer {
 	 */
 	fun onVkSurfaceChanged(surface: Surface, width: Int, height: Int) {
 		GodotLib.resize(width, height)
-		
-		// TODO: properly implement surface re-creation:
-		// Update the native renderer instead of restarting the app.
-		// GodotLib.newcontext should not be called here once it's done.
-		GodotLib.newcontext(surface, false)
+		GodotLib.resetsurface(surface)
```

Comment:
Seems the call order here is important since `DisplayServerAndroid` uses the size set by `GodotLib.resize` here to reset the window.
Would it be simpler to merge the two methods? Or to expand `resetsurface` to also take a `width` and `height` arguments and drop the call to `resize` for the `VkRenderer`?

Reply:
Ok, no problem to merge them.